sqldef_test.go

```diff
@@ -0,0 +1,37 @@
+package sqldef_test
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/k0kubun/sqldef"
+)
+
+func TestParseFiles(t *testing.T) {
```

Comment:
Could you write CLI tests only at a `cmd/*def/*def_test.go` level? For sqldef, the only thing that matters in terms of backward compatibility is CLI, and any functions used for implementing it are just implementation details. When you want to refactor it in a way that modifies `sqldef.ParseFiles`, you will need to modify this test as well, but it'll be an unneeded maintenance cost when you don't need to maintain this function's compatibility.